middleware/setupDB.ts

```diff
@@ -0,0 +1,32 @@
+import { Application, Request, Response, NextFunction } from 'express';
+import { initializeDB } from '../utils/sqliteSetup.js';
```

Comment:
Happy to create a separate ticket for this, but the SQLite DB doesn’t need to be part of MCC or the MOJ template. Shall we remove it from this PR and handle it elsewhere?

Reply:
I'd love to work on that in a separate ticket to minimise the complexity.